platform/linux-generic/arch/aarch64/odp_sysinfo_parse.c

```diff
@@ -336,9 +336,8 @@ int _odp_cpuinfo_parser(FILE *file, system_info_t *sysinfo)
 			if (sysinfo->cpu_hz_max[id] == 0) {
 				uint64_t hz = sysinfo->default_cpu_hz_max;
 
-				_ODP_PRINT("WARN: cpu[%i] uses default max "
-					  "frequency of %" PRIu64 " Hz from "
-					  "config file\n", id, hz);
+				_ODP_WARN("CPU[%i] uses default max frequency of %" PRIu64 ""
+					  "Hz from config file\n", id, hz);
```

Comment:
Whitespace before "Hz" so it's then same as before.

Reply:
Will be fixed on V3.